test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1

```diff
@@ -895,6 +895,20 @@ dir -Recurse `
             $res.CompletionMatches | Should -HaveCount 33
             $res.CompletionMatches[0].CompletionText | Should -BeExactly "Commands"
         }
+
+        It "Test completion with common parameters" {
+            $inputStr = 'invoke-webrequest -out'
+            $res = TabExpansion2 -inputScript $inputStr -cursorColumn $inputStr.Length
+            $res.CompletionMatches | Should -HaveCount 3
+            [string]::Join(',', ($res.CompletionMatches.completiontext | Sort-Object)) | Should -BeExactly "-OutBuffer,-OutFile,-OutVariable"
+        }
+
+        It "Test completion with exact match" {
+            $inputStr = 'get-content -wa'
+            $res = TabExpansion2 -inputScript $inputStr -cursorColumn $inputStr.Length
+            $res.CompletionMatches | Should -HaveCount 4
+            [string]::Join(',', ($res.CompletionMatches.completiontext | Sort-Object)) | Should -BeExactly "-wa,-Wait,-WarningAction,-WarningVariable"
```

Comment:
WarningVariable is a common variable too.

Reply:
Yes, that is included in this validation.  Not sure what you are referring to.

Reply:
Sorry, I meant previous test "Test completion with common parameters" - looks like duplicate.

Reply:
I see.  This test is to ensure that even if provided an exact match `-wa` is an alias to `-warningaction` so it's an exact match, that you still get other completions.  Currently, if you do `get-content -wa`, you only get `-wa` and `-warningaction` and not the others.